develop-guide/writeblitz/writeblitz.rst

```diff
@@ -398,17 +417,17 @@ and later on use that values in other actions
               target: standby
               timeout: 45
               save:
-                - variable_name: first_cmd
-                  filter: contains('pwd')
                 - variable_name: second_cmd
                   filter: contains('ls')
                 - variable_name: everything
               commands:
                 - pwd
                 - ls
                 - |
-                  cd ~  
+                  cd ~
                   echo A string of text
+              include: 
+                  - contains('ls')
```

Comment:
ls is a linux command,  

what about cat file.txt

and contains('some text which is contained in file.txt')

Reply:
to search within that file is not supported, as its looks into this just like a dict. They can do contains('text in the file', regex=True), but that is on its own is limited. I will address a better solution for future

Reply:
Then what does include search for? Not what it prints the screen  just the commadn you send?

Reply:
A dict contains of a pair of command and output to that command. all of them are stored to a dict, include/exclude would treat it like a dict.